shipcat_definitions/src/structs/metadata.rs

```diff
@@ -107,6 +108,10 @@ pub struct Metadata {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub docs: Option<String>,
     // TODO: generate swagger docs url from region and service name
+
+    /// Custom metadata, keys defi
```

Comment:
abbreviated word

Reply:
```suggestion
    /// Custom metadata, keys defined in shipcat.conf
```

I got distracted part way through.